imageroot/actions/configure-module/90certificate

```diff
@@ -20,31 +20,24 @@
 # along with NethServer.  If not, see COPYING.
 #
 
-# Remove traefik route
-
-import os
 import sys
-import json
-import agent
 import agent.tasks
+import os
+import agent
 
-# Try to parse the stdin as JSON.
-# If parsing fails, output everything to stderr
-data = json.load(sys.stdin)
-
-# Find default traefik instance for current node
-default_traefik_id = agent.resolve_agent_id('traefik@node')
-if default_traefik_id is None:
-    sys.exit(2)
+agent_id = agent.resolve_agent_id('traefik@node')
+mail_hostname = os.environ['MAIL_HOSTNAME']
 
-# Remove traefik route
-response = agent.tasks.run(
-    agent_id=default_traefik_id,
-    action='delete-route',
+agent.tasks.run_nowait(
```

Comment:
This task runs in background, but the sync flag is set to true.
What is for? Just to have the notification inside the UI?

Reply:
Yes, even if the `sync` argument is still ignore, I already added it, as @Amygos is planning to implement the sync behavior. From the UI side the task should be visible as running until the certificate is obtained or an error occurs on that side.